package.json

```diff
@@ -24,6 +24,9 @@
     "vue-router": "^3.5.4",
     "vue-simple-spinner": "^1.2.10"
   },
+  "resolutions": {
```

Comment:
What is this for? I had a look at the package and I am not entirely sure what it is doing. Would you be able to add a comment to explain it's usage?

Reply:
Added @achrinza/node-ipc@10.1.9(which already exists as part of the package dependency tree) to package.json
resolutions to fix the version incompatibility issue with node. Added this comment to README.